flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/SequenceFileSerializerFactory.java

```diff
@@ -45,7 +45,7 @@ static SequenceFileSerializer getSerializer(String formatType,
     try {
       type = SequenceFileSerializerType.valueOf(formatType);
     } catch (IllegalArgumentException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)